nodejs/webrtc-hello-again/simple_page.html

```diff
@@ -0,0 +1,166 @@
+<html>
+    <head>
+        <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+        <script src="BandwidthRtc.bundle.js"></script>
+        <script language="javascript">
+            const basePath = 'http://localhost:3000'
+            const bandwidthRtc = new BandwidthRtc();
+            
+            bandwidthRtc.onStreamAvailable((rtcStream) => {
+                console.log("receiving audio!");
+                remoteVideoComponent = document.getElementById("mediaPlayer")
+                remoteVideoComponent.srcObject = rtcStream.mediaStream
+            });
+            bandwidthRtc.onStreamUnavailable((endpointId) => {
+                remoteVideoComponent = document.getElementById("mediaPlayer")
+                console.log("no longer receiving audio");
+                remoteVideoComponent.srcObject = undefined
+
+                // update the interface
+                if(document.getElementById('endButton').disabled){
+                    alert("Call ended, stream is unavailable")
+                }
+
+                setActive();
+                disableButton('endButton')
+                enableButton('callButton')
+            });
+
+            /**
+             * Get the token required to auth with the media server
+             */
+            function getToken(){
+                // prevent double clicks
+                disableButton('onlineButton')
+                console.log("Fetching token from server")
+                const url=basePath + '/startBrowserCall';
+                fetch(url)
+                .then(data=>{return data.json()})
+                .then(res=>{
+                    console.log(res)
+                    startStreaming(res.token)
+                })
+                .catch(error => {
+                    console.error('Error in startCall:', error);
+                });
+            }
+
+            /**
+             * Now that we have the token, we can start streaming media
+             */
+            function startStreaming(token){
+                console.log('connecting to BAND WebRTC server')
+                // Connect to Bandwidth WebRTC
+                bandwidthRtc
+                    .connect({
+                        deviceToken: token,
+                    })
+                    .then(async () => {
```

Comment:
Here you're using `async` (which I like), but elsewhere you're chaining `.then()` statements. We should probably be consistent. I usually prefer `async`/`await` for readability and reducing nesting/indentation.